tekton/ci-lint.yaml

```diff
@@ -1,33 +1,71 @@
 apiVersion: tekton.dev/v1alpha1
 kind: Task
 metadata:
-  name: golangci-lint
+  name: lint
 spec:
   inputs:
     params:
     - name: package
       description: package (and its children) under test
       default: github.com/tektoncd/pipeline
-    - name: flags
+    - name: golangci-flags
       description: flags to use for the lint command
       default: --verbose
-    - name: version
+    - name: golangci-version
       description: golangci-lint version to use
       default: v1.16
     resources:
     - name: source
       type: git
       targetPath: src/${inputs.params.package}
   steps:
-  - name: lint
-    image: golangci/golangci-lint:${inputs.params.version}
+  - name: golangci-lint
+    image: golangci/golangci-lint:${inputs.params.golangci-version}
     workingdir: /workspace/src/${inputs.params.package}
     command:
     - /bin/bash
     args:
     - -c
-    - "golangci-lint run ${inputs.params.flags}"
+    - "golangci-lint run ${inputs.params.golangci-flags}"
     env:
     - name: GOPATH
       value: /workspace
+  - name: header-lint
+    image: bash
+    workingdir: /workspace/src/${inputs.params.package}
+    command:
+      - bash
+    args:
+      - -ce
+      - |
+        header='/*
+        Copyright 2019 The Tektocmd/entrypoint/main.go Authors
+
+        Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+        */'
+
+        ok=true
+        files=`find . -name "*.go" ! -path "./vendor/*"`
+        for file in $files; do
+          content=`cat $file`
+          if [[ ! "$content" =~ "$header" ]]; then
+            echo "$file does not contain the right Copyright heaedr"
```

Comment:
NIT: s/heaedr/header

Reply:
NIT: it would be nice to have the header in a file, and point to it in the error message